recipe/yum_requirements.txt

```diff
@@ -1,6 +1,6 @@
 csh
 imake
-libx11-devel
+libX11-devel
```

Comment:
That makes sense, and after running the re-render command I can see that ci_support/run_docker_build.sh has changed.